### ISSUE TITLE
Train on voc07+12+coco dataset mAP problem

### ISSUE THREAD

**Issue by ujsyehao** [Robert-JunWang/Pelee#35]
Hi, I use your upload coco model, its mAP is 38.3(IOU = 0.5).
Then I finetune the model on voc0712 dataset, I set some parameters below: 

- I follow upload voc0712+coco model's min_size and max_size, ex: (21.28, 45.6)、(45.6, 100.32)、(100.32、155.04)、(155.04, 209.76)、(209.76, 264.48)、(264.48, 319.2)

- I rename all confidence layers, preserve all loc layers.

- I set train batch size 20 and test batch size 4 (due to my GPU memory cost)

- Other hyper-parameters(which in solver.prototxt), I keep it equal to yours.

**Comment by ujsyehao** [comment]
Hi

**Comment by ujsyehao** [comment]
get 75.5% mAP.

**Comment by Robert-JunWang** [comment]
To fine-tune coco model on voc0712, you should use a smaller learning rate. You can modify the parameters based on my solver file.
https://github.com/Robert-JunWang/Pelee/blob/master/model/voc/solver_merged.prototxt

**Comment by ujsyehao** [comment]
@Robert-JunWang thank you!
